packages/bus/src/Brokers/RMQEventBus.ts

```diff
@@ -0,0 +1,395 @@
+import { ExponentialBackoffRetryStrategy } from '../RetryStrategies';
```

Comment:
Use consistent folder naming

```suggestion
import { ExponentialBackoffRetryStrategy } from '../retry-strategies';
```